CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ SET (CMAKE_CXX_EXTENSIONS			OFF	)
 
 IF (NOT WIN32 OR NOT MSVC)
 	ADD_DEFINITIONS ("-Wall -Wextra -Werror -Wno-unused-parameter -Wshadow")
+ELSE()
+	ADD_DEFINITIONS("/Za")
```

Comment:
Can you swap this out for `/permissive-` instead?  The `/permissive-` option is the default in new VS projects, and isn't deprecated like `/Za`.  Thanks!

Reply:
See:  https://docs.microsoft.com/en-us/cpp/build/reference/permissive-standards-conformance?view=msvc-170

Reply:
That also works, done! 😄